src/status_im/contexts/wallet/events.cljs

```diff
@@ -439,3 +439,32 @@
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
                 :duration 10000}]]])})))
+
+(rf/reg-event-fx
+ :wallet/save-address
+ (fn [_ [{:keys [address name color-id on-success on-error chain-short-names ens is-test]
+         :or   {on-success        (fn [])
+                on-error          (fn [])
+                name              ""
+                ens               ""
+                is-test           false
```

Comment:
should be `test?` , no? :)

Reply:
I was trying to be consistent with Go RPC, but `test?` makes more sense.